ui/src/router/components/configuration/components/experiment_config_section/credentials/CredentialsConfigSection.scss

```diff
@@ -1,13 +1,5 @@
 .euiPanel--configSection.experimentCredentials {
   .euiDescriptionList {
     margin-left: 10px;
-
-    .euiDescriptionList__title {
-      width: 70px;
-    }
-
-    .euiDescriptionList__description {
-      width: calc(100% - 70px);
-    }
```

Comment:
These styles have been removed in favour of keeping custom CSS files to a minimum. Their values have been converted to a `columnWidths` prop value in https://github.com/caraml-dev/turing/pull/384/files#diff-32d0bb95d05821ac7eb5f524d7e6f54914e28f5234ea1b5be700af44490f7077R44.

Reply:
not letting elastic do its own sizing often leads to wrong sizing in some case anyway

Reply:
Yeah that's true, I don't know why we wanted to specify such manual sizing in the first place but I guess we can come back again and clean up the entire UI for hardcoded sizes.